content/get-started/transactions.mdx

```diff
@@ -34,8 +48,8 @@ Find the transaction on https://explorer.ironfish.network/transaction/ee32e78268
 
 You can also send in non interactive mode by providing flags. Run `ironfish wallet:send --help` to see the different options available. [See more here](/use/get-started/wallet-commands#send-a-transaction).
 
-## Receive a transaction
-If you want to receive a transaction, you need to give the sender your public address. You can see your public address by running the following command:
+## Receive a transactions
```

Comment:
nit
```suggestion
## Receive transactions
```

Reply:
I put `Receive a transaction` to match the grammar of some of the other titles but this is good too.